### ISSUE TITLE
NRE when loading existing patch

### ISSUE THREAD

**Issue by Xymanek** [AkiniKites/AloysAdjustments#28]
```
System.NullReferenceException: Object reference not set to an instance of an object.
   at AloysAdjustments.Logic.HzdCoreExtensions.GetTypes[T](HzdCore core)
   at AloysAdjustments.Plugins.Outfits.OutfitsGenerator.GetPlayerModels(HzdCore core)
   at AloysAdjustments.Plugins.Outfits.OutfitsGenerator.GenerateModelList(Func`2 coreGetter)
   at AloysAdjustments.Plugins.Outfits.CharacterGenerator.<GetVariantMapping>d__12.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at AloysAdjustments.Plugins.Outfits.OutfitsControl.<LoadPatch>d__39.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at AloysAdjustments.MainWindow.<LoadExistingPack>d__33.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at AloysAdjustments.MainWindow.<InitializePlugins>d__27.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at AloysAdjustments.MainWindow.<Main_Load>d__24.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at AloysAdjustments.UI.Relay.<To>d__1`2.MoveNext()
```

Happened with both 1.7.3 and 1.7.4. Patch was generated using 1.7.3 (I think).

Changes in the existing patch:

* 10x capacity
* TFW outfits remapped to corresponding base game visuals

**Comment by Xymanek** [comment]
I deleted the old patch, created a new one (using 1.7.4) and upon reopening AA, same NRE happens.

What's weird is that I definitely recall the loading of existing patches working properly at some point, but I have no idea what changed since (version maybe?)

**Comment by AkiniKites** [comment]
Could be a version think. I'm trying to maintain compatibility but its getting tricky. Are you still having issues with it?

**Comment by Xymanek** [comment]
Just tried both 1.7.4 and the recent 1.8.0 - both show the same error

**Comment by alxkum** [comment]
Getting the same error here on a fresh install of HZD 1.11 (Steam). I open the app, change the Nora Brave outfit to Carja Blazon Light -> Install Pack, close app, open app again and the error shows up in the status bar with the error.log having the same stack trace as OP posted. As a matter of fact, trying to actually equip the Nora Brave outfit in the game leads to an endless loading icon without ever finishing. No other mods installed.

**Comment by Xymanek** [comment]
> trying to actually equip the Nora Brave outfit in the game leads to an endless loading icon without ever finishing

Works fine here (also 1.11 from steam), although I had to recreate the patch file after the game's update (otherwise the game wouldn't even play the logo sequence)

**Comment by alxkum** [comment]
lol, how does it work for you but not for me? Can you try my patched file? This way we know if it's an issue with the generated file or with my game for whatever reason. This is the file where Nora Brave is replaced with the Carja Blazon Light outfit, please try to equip it. And see if you also get the same error in the app here when that file is in the Packed_DX12 folder.
[Patch_AloysAdjustments.zip](https://github.com/AkiniKites/AloysAdjustments/files/7717629/Patch_AloysAdjustments.zip)

**Comment by Xymanek** [comment]
Tried your file:

* Game works fine (nora brave is correctly remapped) - I tried switched between several armours and nothing went wrong
* App (as expected) throws the same error when trying to load your file (same with mine)

**Comment by alxkum** [comment]
Thanks. Then it's just plain weird for me. I change the outfit in Inventory and when I hit escape to go back into the game, I'm just stuck in the Inventory with the loading spinner in the corner of the screen forever. And when I equip the outfit, quit the game, add the patch, and then try to load the game now - the loading screen will just get stuck at like 80%. In both cases, one core on my CPU is at full load until I kill the process in task manager.

**Comment by AkiniKites** [comment]
This should be fixed in 1.8.1
